tagger/interrogator.py

```diff
@@ -94,13 +99,13 @@ def load(self):
     def unload(self) -> bool:
         unloaded = False
 
-        if hasattr(self, 'model') and self.model is not None:
-            del self.model
+        if self.model is not None:
+            self.model = None
             unloaded = True
+            gc.collect()
```

Comment:
what exactly does gr.collect() do?

Reply:
You may already notice, this line is `gc.collect()` rather than `gr.collect()`. I guess PR author's original purpose is to call garbage collector(gc) to run a full collection. But as we already known, he/she did abandon this approach due to the underlying lib bugs.

Reply:
The new solution looks cool, please give it a try.